lib/internal/fs/rimraf.js

```diff
@@ -219,6 +219,12 @@ function _unlinkSync(path, options) {
           i < tries &&
           options.retryDelay > 0) {
         sleep(i * options.retryDelay);
+      } else {
+        // The file is already gone.
+        if (err.code === 'ENOENT') {
+          return;
+        }
+        throw err;
```

Comment:
nit: we could avoid nested `if` here.
```suggestion
      } else if (err.code !== 'ENOENT') {
        throw err;
      } else {
        // The file is already gone.
        return;
```